lib/phoenix_live_view/test/client_proxy.ex

```diff
@@ -85,6 +85,12 @@ defmodule Phoenix.LiveViewTest.ClientProxy do
       topic: Phoenix.LiveView.Utils.random_id()
     }
 
+    # We build an absolute path to any relative
+    # static assets through the root LiveView's endpoint.
+    static_path =
+      endpoint.config(:otp_app)
+      |> Application.app_dir("priv/static")
```

Comment:
I expect this may be a bit controversial. Very open to suggestions.

Reply:
I think ti is fine for this PR. If someone requests, we can make it configurable.